packages/plugins/package.json

```diff
@@ -41,7 +41,7 @@
     "node-fetch": "^2.6.7",
     "ora": "^5.4.1",
     "semver": "^7.3.7",
-    "solc": "^0.8.17"
+    "yesno": "^0.4.0"
```

Comment:
Added a convenient utility package for handling confirmations. 
https://www.npmjs.com/package/yesno

Also Solc should not be in the plugins package so I've removed it.